ArticleTemplates/simpleBodyTemplatePrepopHelp.html

```diff
@@ -38,11 +38,9 @@ <h2 id="contact">Getting in touch</h2>
             <h2>Subscriptions</h2>
             <ul>
                 <li><a href="#subs-01" data-scroll>What do I get with a premium tier subscription?</a></li>
-                <li><a href="#subs-02" data-scroll>How much does a subscription cost?</a></li>
-                <li><a href="#subs-03" data-scroll>I had an old subscription, will this still work?</a></li>
-                <li><a href="#subs-04" data-scroll>How do I cancel my subscription?</a></li>
-                <li><a href="#subs-05" data-scroll>Can I use my subscription on more than one device?</a></li>
-                <li><a href="#subs-06" data-scroll>I have a print+ Digital, or Guardian subscription. Can I use it to get access to the premium tier?</a></li>
+                <li><a href="#subs-02" data-scroll>How do I cancel my subscription?</a></li>
+                <li><a href="#subs-03" data-scroll>Can I use my subscription on more than one device?</a></li>
+                <li><a href="#subs-04" data-scroll>I have a print+ Digital, or Guardian subscription. Can I use it to get access to the premium tier?</a></li>
```

Comment:
Should "print+ Digital" be "Print+Digital" now, or something else? (Also, is that comma really needed? 🤔)
cc @PriscillaAlcalde

Reply:
👍